packages/mini-browser/package.json

```diff
@@ -4,6 +4,7 @@
   "description": "Theia - Mini-Browser Extension",
   "dependencies": {
     "@theia/core": "^0.6.0",
+    "@theia/editor": "^0.6.0",
```

Comment:
@kittaakos are you against adding `@theia/editor` as a dependency to the mini-browser?
In this case it is used to open a uri with the `editor-handler` in order to properly implement the `Open Source` method (since I believe opening the source always means with the editor).